exercises/lyapunov/control/BUILD.bazel

```diff
@@ -0,0 +1,14 @@
+# -*- mode: python -*-
+# vi: set ft=python :
+
+# Copyright 2020 Massachusetts Institute of Technology.
+# Licensed under the BSD 3-Clause License. See LICENSE.TXT for details.
+
+load("//tools/rt/jupyter:defs.bzl", "rt_ipynb_test")
+
+rt_ipynb_test(
+    name = "control",
+    srcs = ["control.ipynb"],
+    size = "medium",
```

Comment:
do you know that these need size=medium?  In general, we might not want to produce too many long-running tests....

(otherwise this PR looks good)